src/ControlSystem/UpdateFunctionOfTime.cpp

```diff
@@ -44,4 +50,100 @@ void ResetFunctionOfTimeExpirationTime::apply(
     const std::string& f_of_t_name, const double new_expiration_time) {
   (*f_of_t_list).at(f_of_t_name)->reset_expiration_time(new_expiration_time);
 }
+
+UpdateAggregator::UpdateAggregator(
+    std::unordered_set<std::string> control_system_names)
+    : individual_names_(std::move(control_system_names)),
+      combined_name_([this]() {
```

Comment:
This makes an assumption that all control systems are active, which may not be the case. Rather than taking all control system names and then creating the combined name, pass the combined name and only the active control systems:

```cpp
UpdateAggregator::UpdateAggregator(
     std::string combined_name,
     std::unordered_set<std::string> active_control_system_names)
```

This will require the control system Initialization action (where this class gets constructed, in #5378) to know about all active/inactive control systems. Currently, that information is only available at option parsing. I will have to make a global cache tag to expose that information.